src/app/settings/panel-settings/components/edit-panel-configuration-dialog/edit-panel-configuration-dialog.component.html

```diff
@@ -0,0 +1,3 @@
+<sofie-sidebar-container [title]="getTitle()">
+    <sofie-edit-panel-configuration [panelConfiguration]="panelConfiguration" (onSave)="closeDialog($event)" (onCancel)="closeDialog()"></sofie-edit-panel-configuration>
```

Comment:
```suggestion
    <sofie-edit-panel-configuration [panelConfiguration]="panelConfiguration" (onSave)="closeDialog($event)" (onCancel)="closeDialog()" />
```
Just a suggestion :)

Reply:
I'm a fan of self-closing tags.
I just remember that not so long ago we weren't able to use self-closing tags on our own components. I can't remember why and I am positive that at one of my previous workplaces we did use it...

It's been updated :)

Reply:
Angular 16 brings us this joy :-D